delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -17,6 +17,12 @@
 _compile_itemnum = re.compile(r'[^0-9]')
 
 
+def _sanitize_string(string):
+    if not string:
+        return False
```

Comment:
is the original string a boolean in this case? I think it makes more sense to return `""`